quick-start/src/main/ui/app/shared/stories/components/new-entity/new-entity.component.stories.ts

```diff
@@ -0,0 +1,65 @@
+import {moduleMetadata, storiesOf} from '@storybook/angular';
+import {object, text, boolean, withKnobs} from '@storybook/addon-knobs';
+import {withNotes} from '@storybook/addon-notes';
+import {action} from '@storybook/addon-actions';
+import {linkTo} from '@storybook/addon-links';
+import {centered} from '@storybook/addon-centered/angular';
+import {StoryCardComponent} from '../../utils/story-card/story-card.component';
+import {NewEntityComponent} from '../../../components/new-entity/new-entity.component';
+import {ThemeModule} from '../../../components/theme/theme.module';
+import {MdlDialogService} from '@angular-mdl/core';
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-dialog-button',
+  template: '<button (click)="openModal()">Open Modal</button>'
+})
+export class DialogButtonComponent {
+  @Output() createClicked = new EventEmitter();
+  constructor(
+    private dialogService: MdlDialogService
+  ) { }
+  openModal() {
+    this.dialogService.showCustomDialog({
+      component: NewEntityComponent,
+      providers: [
+        { provide: 'actions', useValue: {save: () => {
+          this.create();
```

Comment:
No need to create the function here, just execute:
```suggestion
           this.createClicked.emit()
```